jina/proto/jina.proto

```diff
@@ -314,6 +314,7 @@ message RequestProto {
     message DataRequestProto {
         repeated DocumentProto docs = 1; // a list of Documents to query
         repeated DocumentProto groundtruths = 2; // a list of groundtruth Document you want to evaluate it with
+        string pod_name = 3; // sender pod name
```

Comment:
do not add this here, why is this needed?

Reply:
https://github.com/jina-ai/jina/pull/4030#discussion_r767439206

Reply:
removed